zcbor/zcbor.py

```diff
@@ -1665,9 +1667,13 @@ def child_declarations(self):
 
     # Declaration of the variables of all children.
     def child_single_declarations(self):
-        decl = [
-            line for child in self.value for line in child.add_var_name(
-                child.single_var_type(), anonymous=True)]
+        decl = []
+        for child in self.value:
+            if child.is_unambiguous_repeated():
+                pass
```

Comment:
Why do we need this check if it simply passes?

Reply:
Changed